src/main/java/com/jw/exception/ExceptionMessages.java

```diff
@@ -0,0 +1,12 @@
+package com.jw.exception;
+
+public class ExceptionMessages {
```

Comment:
Since that this class is an "Utility/constant" class, you can or change to interface OR (which imho is better) use `@UtilityClass` from lombok, it'll create the private constructor, and set all field as static, so you dont need to use static in here